src/index.html

```diff
@@ -9,6 +9,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header-big">
+      <nav class="nav-big">
+        <img src="images/logo.png" alt="downloading..." class="logo">
+        <ul class="list">
+          <li><a href="#" class="list-active is-active">APPLE</a></li>
```

Comment:
please, use indents to separate your code blocks
```suggestion
          <li>
            <a href="#" class="list-active is-active">
              APPLE
            </a>
          </li>
```